frontend/src/components/SyncEditor/SyncEditor.test.tsx

```diff
@@ -1319,7 +1330,7 @@ const semanticErrors = [
     options: {
       isWholeLine: true,
       glyphMarginClassName: 'errorDecoration',
-      glyphMarginHoverMessage: { value: "```html\nMust have required property 'name' \n```" },
+      glyphMarginHoverMessage: { value: "```html\nmust have required property 'name' \n```" },
```

Comment:
@jeswanke Why did you change these? I didn't have a problem with the error messages starting with a capital letter in general - it was just when you were using user input that did not match what the user had entered in the YAML editor.